.github/workflows/build.yml

```diff
@@ -105,6 +105,8 @@ jobs:
         if: matrix.dependencies != ''
         run: |
           sudo apt-get update -y && sudo apt-get install -y ${{ matrix.dependencies }}
+          # Work around lldb on Ubuntu 22.04 issue https://github.com/llvm/llvm-project/issues/55575
+          sudo  ln -s /usr/lib/llvm-14/lib/python3.10/dist-packages/lldb/* /usr/lib/python3/dist-packages/lldb/
```

Comment:
Is there any way you could avoid hard-coding `llvm-14` and `python-3.10` and retrieve the LLVM and Python versions from the system instead?

Reply:
unless we expect this bug to be fixed for upcoming llvm/python3 releases and using this work-around would actually become a problem on its own

Reply:
The bug is fixed in 24.04 LTS so this is a temporary work around until we update CI to use that version. Based on https://github.com/actions/runner-images/issues/10636 that will be between now and the October 30 2024. At which point this will break and we can remove it.